static-checks/nist-validation/test.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python3
+
+import re
+import requests
+from pathlib import Path
+
+from lib import util, results
+
+
+# Get scapval zip file link from NIST web
+scapval_web = 'https://csrc.nist.gov/Projects/Security-Content-Automation-Protocol/SCAP-Releases/SCAP-1-3' # noqa
+r = requests.get(scapval_web)
+scapval_web_path = re.search(r'href="(.*scapval-.*.zip)"', r.text)
+if not scapval_web_path:
+    raise RuntimeError("SCAPval tool link not found on NIST web")
+scapval_link = 'https://csrc.nist.gov' + scapval_web_path.group(1)
+scapval_zip = scapval_link.split('/')[-1]
+
+with util.get_content(build=False) as content_dir:
+    util.subprocess_run(['wget', scapval_link], check=True)
+    util.subprocess_run(['unzip', scapval_zip], check=True)
```

Comment:
Doing this in python isn't as difficult as you think:
```python
import io
import zipfile
import requests

url = 'https://github.com/RHSecurityCompliance/contest/archive/refs/heads/main.zip'
reply = requests.get(url)
reply.raise_for_status()
zip = zipfile.ZipFile(io.BytesIO(reply.content))
zip.extractall(path='/tmp/foo')
```
where `/tmp/foo` is `tempfile.TemporaryDirectory()`, or omit the argument altogether if you're comfortable relying on CWD being temp dir.

There are more memory-efficient ways of doing it, like chunking the download via
``` 
with requests.get(url, stream=True) as r:
    r.raise_for_status()
    with open(local_filename, 'wb') as f:
        for chunk in r.iter_content(chunk_size=8192): 
            # write to some Bytes stream from which zipfile can read
```
but 48 MB is probably well within the memory capabilities of any system we run this suite on.

I think I would prefer the short python version over `wget` / `unzip`, but I leave this up to you.